docs/content/documentation/systems/AuxKernels/tensor_mechanics/RankTwoScalarAux.md

```diff
@@ -1,10 +1,232 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
-# RankTwoScalarAux
+# Rank Two Scalar Aux
 !syntax description /AuxKernels/RankTwoScalarAux
 
+##Description
+The AuxKernel `RankTwoScalarAux` provides methods to calculate several different scalar quantities for a Rank-2 tensor, described below.
+In some types of calculations, the scalar quantity is calculated in a user-specified direction (by default set to the (0, 0, 1) axis); in other calculation types the user can specify the start and end points of a line along which the scalar quantity is calculated.
+
+If desired, `RankTwoScalarAux` can be restricted to calculate the scalar quantity data for a Rank-2 tensor at a single specified quadrature point per element. This option is generally used only for debugging purposes.
+
+As with the [RankTwoAux](/RankTwoAux.md) AuxKernel, `RankTwoScalarAux` requires the inclusion of an AuxVariable block for each AuxKernel block.
+!!! note "Elemental vs Nodal Visualization of Quadrature Field Values"
+    Using an AuxVariable setting of `order = CONSTANT` will give a constant value of the AuxVariable for the entire element. Using an AuxVariable argument of `order = FIRST` or higher will cause libMesh to output the AuxVariable as a nodal variable for visualization purposes; this higher monomial variable setting can produce smoother results for a properly converged simulation.
```

Comment:
Perhaps this note should go at the bottom of the page, just above the InputParameters section, instead of here?  Since we've had some recent users-lists questions about this subject, I wanted to copy @bwspenc's answer into our documentation.